enclave/crypto/openssl/CMakeLists.txt

```diff
@@ -24,9 +24,6 @@ maybe_build_using_clangw(oecryptoopenssl)
 
 enclave_enable_code_coverage(oecryptoopenssl)
 
-enclave_include_directories(oecryptoopenssl PUBLIC
-                            $<BUILD_INTERFACE:${OE_INCDIR}/openenclave>)
-
 enclave_include_directories(
```

Comment:
For my understanding purpose, can you explain what we are doing here earlier and now?

Reply:
This is just a duplicated include. When linking against `openssl`, `oecryptoopenssl` will inhere the same include path.